standards/python.md

```diff
@@ -37,13 +37,13 @@ We use [isort](https://github.com/timothycrosley/isort) to lint imports to compl
 
 The isort configuration can go into a `tox.ini` file under an `[isort]` header.
 
-## Python versions
+## Python version
 
-As we transition from Python 2.7 to Python 3 it is highly recommended to test against both Python 2.7 and Python 3.6.
+It is highly recommended to test against Python 3.6.
```

Comment:
Once we're on 3.6, couldn't we simplify to this?

We test against Python 3.6.

Reply:
Great suggestion. I made the minor update you suggested.